pages/setup/account.page.tsx

```diff
@@ -19,7 +22,7 @@ import {
   AccountListOptionsQuery,
 } from './Account.generated';
 
-type AccountList = AccountListOptionsQuery['accountLists']['nodes'][number];
+export type AccountList = AccountListOption;
```

Comment:
You don't need to create a new type called `AccountList` as you can just use `AccountListOption`
```suggestion
```